src/commands/backup/schedule/create.tsx

```diff
@@ -0,0 +1,100 @@
+import { ExecRenderBaseCommand } from "../../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../rendering/process/process_flags.js";
+import { ReactNode } from "react";
+import { Flags } from "@oclif/core";
+import { assertStatus } from "@mittwald/api-client-commons";
+import { Success } from "../../../rendering/react/components/Success.js";
+import { Value } from "../../../rendering/react/components/Value.js";
+
+import { projectFlags } from "../../../lib/resources/project/flags.js";
+
+type Result = {
+  projectBackupScheduleId: string;
+};
+
+export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
+  static summary = "Create a new backup schedule";
+  static flags = {
+    ...projectFlags,
+    ...processFlags,
+    description: Flags.string({
+      description: "Set the description for the backup schedule",
+    }),
+    schedule: Flags.string({
+      required: true,
+      description: "Define the schedule itself",
+    }),
+    ttl: Flags.string({
+      required: true,
+      description:
+        "Define the backup storage period in days, for through this schedule created backups",
```

Comment:
From what I read in the (albeit not very thoroughly documented) API spec[^1], this is _not_ just a "number of days", but a generic interval string, in which the user can specify the unit themselves (so for example, `12d`, `3h` or `3w`). This should be documented accordingly.

Also, check if you can use the `Duration` class,[^2] which also offers a specialized flag type (its `toString` method might need to be extended though, to fit this use case better).

[^1]: https://developer.mittwald.de/docs/v2/reference/backup/backup-create-project-backup-schedule/
[^2]: https://github.com/mittwald/cli/blob/master/src/lib/units/Duration.ts

Reply:
I was playing around with the values but anything except "[int]d" yields the response 
```
 "type": "ValidationError",
    "message": "3 INVALID_ARGUMENT: ttl format 7 is not allowed, allowed is '{days}d' e.g. '7d', '30d'",
    "validationErrors": [
```
Now Im sure that we could take on (milli-)seconds as ttl format in true unix fashion, I want to put to discussion though if the pros outweight the cons toreason for implementing it. In my opinion `-ttl=[int]` whould be all a user has to do for the ttl.

there are even hard limits in place:
```
"type": "ValidationError",
"message": "3 INVALID_ARGUMENT: ttl 500d with value 500 is not allowed, allowed values are between 7 and 400",
```